models/user.rb

```diff
@@ -10,9 +10,12 @@ class User < Base
 
   RESET_WINDOW = 60 * 15 # 15 minutes
 
-  SETTINGS = %w[
-    notifications bg font bg2 font2 red_logo white yellow green brown gray red blue
-  ].freeze
+  SETTINGS = Array.new(4) do |n|
```

Comment:
why are you make a size 4 array? 

maybe you wanted to do 4.time.flat_map ?

Reply:
I had `4.times.flat_map` there, but stumbled upon a benchmark somewhere stating Array.new(n) is significantly more performant than n.times.map. But flat_map doesn’t need flatten!, so here we go …

Reply:
Is there a way to access Lib::Settings::ROUTE_COLORS here? (… to get rid of hardcoded 4)